### DISCUSSION TITLE
Is it compatible with Spring Boot 3.3.0?

### DISCUSSION THREAD

**Discussion by alexbujenita** [awspring/spring-cloud-aws#1155] [Q&A] [answered]
Hi all,

Trying to upgrade some services to SpringBoot 3.3.0, but wen running the tests the application doesn't start mentioning:

```
Description:

Your project setup is incompatible with our requirements due to following reasons:

- Spring Boot [3.3.0] is not compatible with this Spring Cloud release train
```

In my pom I have

```xml
            <dependency>
                <groupId>io.awspring.cloud</groupId>
                <artifactId>spring-cloud-aws-dependencies</artifactId>
                <version>3.2.0-M1</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>

        <dependency>
            <groupId>io.awspring.cloud</groupId>
            <artifactId>spring-cloud-aws-starter-sqs</artifactId>
        </dependency>
        <dependency>
            <groupId>io.awspring.cloud</groupId>
            <artifactId>spring-cloud-aws-starter</artifactId>
        </dependency>
```

**Comment by atulgupta7** [comment]
https://github.com/awspring/spring-cloud-aws/releases

Seems Spring Cloud AWS is not yet ready for Spring Boot 3.3
@maciejwalkowiak

**Comment by maciejwalkowiak** [comment, marked as answer]
@atulgupta7 @alexbujenita apologies for late reply. While 3.1.3 was already compatible with Spring Boot 3.3, we have released official support for Spring Boot 3.3 in Spring Cloud AWS 3.2.0 two days ago.

**Reply by alexbujenita** [reply]
Yep, saw that and already upgraded some services, thanks!

**Reply by atulgupta7** [reply]
Thanks @maciejwalkowiak 
I am successfully able to use aws spring 3.3 with spring boot 3.3.3
